ruby_on_rails/cucumber.md

```diff
@@ -0,0 +1,86 @@
+# Cucumber
+
+[Cucumber](https://cucumber.io) is a testing framework which allows us to specify feature tests using plain language.
+In our projects, we use Cucumber for e2e tests. We integrate it using the
+`cucumber-rails` gem.
+
+## Installing `cucumber-rails`
+
+The installation process of `cucumber-rails` is also documented on its
+[Github page](https://github.com/cucumber/cucumber-rails).
+
+To install `cucumber-rails`, add the following gems to your Gemfile:
+ 
+```rb
+group :test do
+    gem 'capybara', require: false
+    gem 'cucumber-rails', require: false
+    gem 'database-cleaner'
```

Comment:
I would make this future-ready: system tests and no database cleaner. Is it possible with cucumber to leverage that?

Reply:
I had a quick look, but didn't manage to get it running. But as it is would be nice to have, I'll invest another hour for that.

Reply:
Invested an hour. Did not find a way to integrate it. :-(

Reply:
cucumber does not support system tests yet therefore we need database cleaner. :(